docs/docsite/rst/network_getting_started.rst

```diff
@@ -0,0 +1,246 @@
+.. network-getting-started-example:
+
+*******************************
+Network getting started example
+*******************************
+
+.. contents:: Topics
+
+
+Overview
+========
+
+Objective
+---------
+
+I want to understand how Ansible can be used to connect to multiple network devices.
+
+.. FIXME FUTURE Gundalow - Link to examples index once created
+
+.. note:: Examples, not production code
+   This is not for production use, it's more to demonstrate the different areas that need to be considered and provide a foundation level knowledge.
+
+Prerequisites
+-------------
+
+* Ansible 2.3 (or higher) installed :doc:`intro_installation`
+* One or more network device compatible with Ansible
```

Comment:
Can we have link http://docs.ansible.com/ansible/latest/list_of_network_modules.html to point list of available network modules.

Reply:
Will be a separate page, maybe on `network_into`, or if not, a page under that.